packages/api-client/src/main/tcp/WebSocketClient.ts

```diff
@@ -79,14 +82,23 @@ export class WebSocketClient extends EventEmitter {
     this.baseUrl = baseUrl;
     this.client = client;
     this.hasUnansweredPing = false;
-    this.isOnline = false;
+    this.isRefreshingAccessToken = false;
+    this.shouldSendPing = true;
 
     this.logger = logdown('@wireapp/api-client/tcp/WebSocketClient', {
       logger: console,
       markdown: false,
     });
   }
 
+  private readonly onReconnect = () => {
+    this.logger.info('Reconnecting to WebSocket');
+    // TODO: Remove this hack once `maxEnqueuedMessages` works
```

Comment:
See https://github.com/pladaria/reconnecting-websocket/issues/110

Reply:
Please link the ticket in our code.